config/crd/bases/dbaas.redhat.com_mongodbatlasconnections.yaml

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
can you add an example of the CR for this under config/samples

Reply:
OK